travis.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash
+
+build_cpp() {
+  ./autogen.sh
+  ./configure
+  make -j2
```

Comment:
Historical precedent (the old `.travis.yml`) and the docs [here](http://docs.travis-ci.com/user/speeding-up-the-build/#Parallelizing-your-build-on-one-VM) which specify "Travis CI VMs run on 1.5 virtual cores".